cloudprober/values.yaml

```diff
@@ -20,6 +20,9 @@ image:
 configMap:
   name: ''
   create: true
+  # The config file suffix; supported suffixes are in
+  # https://github.com/cloudprober/cloudprober/blob/master/config/config.go#L72
+  format: 'cfg'
 
```

Comment:
It's currently failing because you're accessing it at .Values.config.format while it's under configMap right now. I'd probably out-indent it and call it `configFormat` (and access it as Values.configFormat).

Reply:
Good point, fixed.